mix.exs

```diff
@@ -29,4 +41,22 @@ defmodule Janus.MixProject do
       {:ex_doc, "~> 0.22", only: [:test, :dev], runtime: false}
     ]
   end
+
+  defp package do
+    [
+      maintainers: ["ElixirJanus Team"],
```

Comment:
I'd rather put Software Mansion here, `ElixirJanus Team` is something that exists only on Github